src/pages/Lending/Pool/components/Repay/RepayInput.tsx

```diff
@@ -626,13 +626,13 @@ export const RepayInput = ({
         assetSymbol={collateralAsset?.symbol ?? ''}
         assetIcon={collateralAsset?.icon ?? ''}
         // Both cryptoAmount and fiatAmount actually defined at display time, see showFiatSkeleton below
-        cryptoAmount={lendingQuoteCloseData?.quoteWithdrawnAmountAfterFeesCryptoPrecision}
-        fiatAmount={lendingQuoteCloseData?.quoteWithdrawnAmountAfterFeesUserCurrency}
+        cryptoAmount={lendingQuoteCloseData?.quoteWithdrawnAmountAfterFeesCryptoPrecision ?? '0'}
+        fiatAmount={lendingQuoteCloseData?.quoteWithdrawnAmountAfterFeesUserCurrency ?? '0'}
         isAccountSelectionDisabled={isAccountSelectionDisabled}
         isSendMaxDisabled={false}
         percentOptions={percentOptions}
         showInputSkeleton={isLendingQuoteCloseLoading || isLendingQuoteCloseRefetching}
-        showFiatSkeleton={false}
+        showFiatSkeleton={isLendingQuoteCloseLoading || isLendingQuoteCloseRefetching}
```

Comment:
can we also add this to the `showFiatSkeleton` above for repayAmount? I thought this was broken because I was looking at the top one

Reply:
Might as well! 969d8d35a